packages/plugins/rubygem-foreman_git_templates/rubygem-foreman_git_templates.spec

```diff
@@ -0,0 +1,101 @@
+# Generated from foreman_git_templates-0.0.1.gem by gem2rpm -*- rpm-spec -*-
+# template: foreman_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_git_templates
+%global plugin_name git_templates
+%global foreman_min_version 1.20
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.0.0
+Release: 1%{?foremandist}%{?dist}
+Summary: Adds support for using templates from Git repositories
+Group: Applications/Systems
+License: GPL-3.0
+URL: https://github.com/dm-drogeriemarkt-de/foreman_git_templates
+Source0: %{gem_name}-%{version}.gem
```

Comment:
This can be a full URL to rubygems.org I guess. A recent change by @evgeni always downloads the source and verifies it doesn't change, but I guess it didn't expect this situation. Luckily this is open source and we can just download it :)

Reply:
This is a leftover from me running gem2rpm on an unreleased version of the plugin. ;-)